acceptance-tests/pom.xml

```diff
@@ -123,7 +123,6 @@
 				<plugin>
 					<groupId>org.apache.maven.plugins</groupId>
 					<artifactId>maven-jar-plugin</artifactId>
-					<version>3.1.0</version>
 					<configuration>
```

Comment:
looks like version was removed, was it intended ? @john-manack

Reply:
@chetanmunegowda yes, this plugin adopts the version from the parent pom.xml file, which contains the version update.